templates/verify_body.html

```diff
@@ -475,7 +475,8 @@
                               <tr>
                                 <td align="center" class="footerInfo" style="font-size:0px;padding:0px;word-break:break-word;">
                                   <div style="font-family:Open Sans, sans-serif;font-size:13px;line-height:1;text-align:center;color:#000000;">{{.Props.QuestionInfo}}
-                                    <a href="mailto:{{.Props.SupportEmail}}"> {{.Props.SupportEmail}}</a>
+                                    <a href="mailto:{{.Props.SupportEmail}}">
+                                      {{.Props.SupportEmail}}</a>
```

Comment:
maybe we want to put the closing anchor tag in a new line?

Reply:
This file is auto-generated

Reply:
gotcha